src/util/clickTracking.test.js

```diff
@@ -0,0 +1,66 @@
+import * as clickTracking from './clickTracking';
+
+/*
+ * **** DISABLED ******
+ *
+ * This is disabled because tests don't have `global.documnet` defined in
```

Comment:
What about travis is different is different than the local node env that would make `global.document` unavailabe?

Also, typo in "document"